app/controllers/provider/invitee_signups_controller.rb

```diff
@@ -49,7 +49,7 @@ def can_create?
   end
 
   def build_user
-    @user = @invitation.make_user(params[:user] || {})
+    @user = @invitation.make_user(params.permit![:user].to_h || {})
```

Comment:
I would prefer having a list of permitted attributes that can be passed there than permit everything

Reply:
@hallelujah Done